test-infra/prow/config.yaml

```diff
@@ -231,6 +231,33 @@ periodics:
        - name: compute-image-tools-test-service-account
          secret:
            secretName: compute-image-tools-test-service-account
+ - name: ci-v2v-adapt-e2e
+   interval: 24h
```

Comment:
This specifies how often it runs? Since Sam is iterating, and the suite is tiny, maybe increase the frequency of runs?

Reply:
Increase to 12hrs for the time being. The test count will match translate test count. We can tweak the frequency later on.